code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -953,6 +953,19 @@
 	reagents.add_reagent("sugar", 3)
 	bitesize = 2
 
+/obj/item/reagent_container/food/snacks/pancakes
+	name = "pancakes"
+	desc = "Golden brown creamy pancakes fresh from the griddle. Drizzled with maple syrup and topped with a slice of butter."
+	icon_state = "pancakes"
+	filling_color = "#bb9857"
+
+/obj/item/reagent_container/food/snacks/pancakes/Initialize()
+	. = ..()
+	reagents.add_reagent("bread", 6)
+	reagents.add_reagent("sugar", 4)
+	reagents.add_reagent("milk", 5)
+	bitesize = 2
```

Comment:
Why is this set on init

Reply:
That shouldn't be how its set then

Reply:
sorry but this is just how it's done, rewriting the entire bit of food code for one pancake PR is waaaay out of scope for this

Reply:
i mean, you can literally just put `bitesize = 2` in the type declaration and that fixes it